layouts/community/ergodox/azurefog/config.h

```diff
@@ -0,0 +1,12 @@
+/*
+  Set any config.h overrides for your specific keymap here.
+  See config.h options at https://docs.qmk.fm/#/config_options?id=the-configh-file
+*/
+#define DISABLE_RGB_MATRIX_DIGITAL_RAIN
```

Comment:
THis should include ... well, an include guard;
```suggestion
#pragma once 

#define DISABLE_RGB_MATRIX_DIGITAL_RAIN
```